packages/fluentui/react-builder/src/components/Designer.tsx

```diff
@@ -374,42 +430,88 @@ export const Designer: React.FunctionComponent = () => {
       <div style={{ display: 'flex', flex: 1, minWidth: '10rem', overflow: 'hidden' }}>
+        <Box
+          styles={({ theme }) => ({
+            background: '#FAF9F8',
+            borderRight: `1px solid ${theme.siteVariables.colorScheme.default.border2}`,
+            display: 'flex',
+            flexDirection: 'column',
+            width: '3.4rem',
+            transition: 'opacity 0.2s',
+            position: 'relative',
+            ...(mode === 'use' && {
+              pointerEvents: 'none',
+              opacity: 0,
+            }),
+          })}
+        >
+          <NavBarItem
+            title="Add components"
+            isSelected={activeTab === 'add'}
+            icon={<AddIcon size="large" outline />}
+            onClickHandler={() => selectActiveTab('add')}
+          />
+
+          <NavBarItem
+            title="Navigator"
+            isSelected={activeTab === 'nav'}
+            icon={<MenuIcon size="large" outline />}
+            onClickHandler={() => selectActiveTab('nav')}
+          />
+        </Box>
         <div
           style={{
             display: 'flex',
             flexDirection: 'column',
-            minWidth: '12rem',
+            minWidth: '22.85rem',
             transition: 'opacity 0.2s',
             ...(mode === 'use' && {
               pointerEvents: 'none',
               opacity: 0,
             }),
           }}
         >
-          <List style={{ overflowY: 'auto' }} onDragStart={handleDragStart} />
-          <div role="complementary" aria-label="Component tree">
-            <Divider style={{ margin: '1rem' }} />
-            {jsonTree?.props?.children?.length > 0 ? (
-              <ComponentTree
-                tree={jsonTree}
-                selectedComponent={selectedComponent}
-                onSelectComponent={handleSelectComponent}
-                onCloneComponent={handleCloneComponent}
-                onMoveComponent={handleMoveComponent}
-                onDeleteSelectedComponent={handleDeleteSelectedComponent}
-                onAddComponent={handleOpenAddComponentDialog}
-              />
-            ) : (
-              <Button
-                text
-                content="Insert first component"
-                fluid
-                onClick={() => handleOpenAddComponentDialog('', 'first')}
-              />
-            )}
+          <div
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              padding: '0 10px 0 20px',
+              borderBottom: '1px solid #E1DFDD',
+            }}
+          >
+            <Header as="h2" style={{ fontSize: '16px', fontWeight: 'bold' }}>
+              {activeTab == 'add' ? 'Add components' : 'Navigator'}
+            </Header>
           </div>
+          {activeTab == 'add' && (
+            <div>
+              <List style={{ overflowY: 'auto' }} onDragStart={handleDragStart} />
+              {jsonTree?.props?.children?.length === 0 && (
+                <Button
```

Comment:
the add button belongs to the nav tab - it allows the screen reader users to start designing - after first component is added, they can use the tree to add after, before or add children. With this, maybe we do not need the add tab at all?